### PR TITLE
Feat: Conversations.info method, test, extensions and mocks

### PR DESCRIPTION
**Pull request by BenDenger**
Added conversations. info method 

## Current Behavior
missing method

## Improvement/New Behavior
added conversations.info method, default implementation, mocks & tests, test extensions

### CODE REVIEW

File: data/slack-jackson-dto/src/main/kotlin/io/olaph/slack/dto/jackson/common/types/Channel.kt [hunk 1]
```diff
@@ -30,7 +30,11 @@ data class Channel(
         @JsonProperty("topic") val topic: Topic,
         @JsonProperty("purpose") val purpose: Purpose,
         @JsonProperty("previous_names") val previousNames: List<String>,
-        @JsonProperty("num_members") val numMembers: Int) {
+        @JsonProperty("num_members") val numMembers: Int,
```
Comment (MarvinSchramm) [inline data/slack-jackson-dto/src/main/kotlin/io/olaph/slack/dto/jackson/common/types/Channel.kt]: this has to be an optional

Comment (BenDenger) [reply data/slack-jackson-dto/src/main/kotlin/io/olaph/slack/dto/jackson/common/types/Channel.kt]: does it really? the field was already included

Comment (MarvinSchramm) [reply data/slack-jackson-dto/src/main/kotlin/io/olaph/slack/dto/jackson/common/types/Channel.kt]: ```When using the method with the include_num_members parameter, we return a num_members field```

https://api.slack.com/methods/conversations.info

**Comment by MarvinSchramm** [comment]
can you also rebase?

**Pull request merged**